src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -297,7 +350,7 @@ public void run() {
                 // @checkstyle MagicNumber (1 line)
                 final byte[] buffer = new byte[4096];
                 for (int count = input.read(buffer); count != -1;
-                    count = input.read(buffer)) {
+                     count = input.read(buffer)) {
```

Comment:
@alexey-krylov I would suggest submit a ticket https://github.com/teamed/xdsd

>  i offer to exchange with settings via https://gitlab.i-free.com/mretail/idea-settings.git plugin. This is default option of IDEA. We can create teamed.io special repo for configs. This will significantly reduce codestyle frustration.